.github/actions/templates/getWorkflowInput/action.yml

```diff
@@ -61,7 +61,15 @@ runs:
 
         # When running from workflow_dispatch event get input values
         if ('${{ github.event_name }}' -eq 'workflow_dispatch') {
-          $removeDeployment='${{ github.event.inputs.removeDeployment }}'
+          $parameters = '${{ toJSON(github.event.inputs) }}' | ConvertFrom-Json -AsHashTable
+
+          # Output values to be accessed by next jobs
+          $workflowInput = @{}
+          foreach($parameterName in $parameters.Keys) {
+            Write-Verbose ('Passing output [{0}] with value [{1}]' -f $parameterName, $parameters[$parameterName]) -Verbose
+            $workflowInput[$parameterName] = $parameters[$parameterName]
+          }
+          Write-Output ('{0}={1}' -f 'workflowInput', ($workflowInput | ConvertTo-Json -Compress)) >> $env:GITHUB_OUTPUT
```

Comment:
How can we make sure this won't cause issues as per bug #2323 ?

Reply:
Because it ist just a single line string and not e.g. an object, or multi line string. Works like a charm